docs/source/02_alfasim_sdk_by_example.rst

```diff
@@ -216,7 +216,7 @@ needed to load and unload the |sdk| API, in which will allows the plugin to use
     }
 
 Then, since the plugin wants to calculate its own secondary variable, the
-:func:`HOOK_UPDATE_PLUGINS_SECONDARY_VARIABLES <alfasim_sdk.hook_specs.update_plugins_secondary_variables>` must be implemented.
+:func:`HOOK_UPDATE_PLUGINS_SECONDARY_VARIABLES <_alfasim_sdk.hook_specs.update_plugins_secondary_variables>` must be implemented.
```

Comment:
This link should use the public interface, rather than reference an internal module...

Reply:
`hook_specs` is not intended to be used by ALFASim-SDK users and for this reason I kept as private, 

Basically `hook_specs` is an input for an internal tool (python-hookman) to generate the header files (hooks_specs.h) for the end-users.